packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -52,9 +53,19 @@ export class DetailsRowFields extends React.PureComponent<IDetailsRowFieldsProps
               ? onRender(item, itemIndex, column)
               : getCellText(item, column);
 
+          // generate a key that auto-dirties when content changes, to force the container to re-render, to trigger animation
+          let key: number | string;
+          try {
+            const s = JSON.stringify(cellContentsRender);
```

Comment:
woah. Please do not call JSON.stringify in this super duper hot spot perf place.